packages/screens/Governance/GovernanceProposal/GovernanceVoteHeader/components/VoteTimeDetailsBox.tsx

```diff
@@ -46,19 +48,41 @@ export const VoteTimeDetailsBox: React.FC<{
           label="Deposit End Time"
           subLabel={`${deposit_end_time.slice(0, 10)} ${deposit_end_time.slice(11, 16)} UTC`}
         />
-        <Separator horizontal />
-
-        <VoteTimeText
-          label="Voting Start"
-          subLabel={`${voting_start_time.slice(0, 10)} ${voting_start_time.slice(11, 16)} UTC`}
-        />
-        <Separator horizontal />
-
-        <VoteTimeText
-          label="Voting End"
-          subLabel={`${voting_end_time.slice(0, 10)} ${voting_end_time.slice(11, 16)} UTC`}
-        />
+        {!isMobile && (
+          <>
+            <Separator horizontal />
+            <VoteTimeText
+              label="Voting Start"
+              subLabel={`${voting_start_time.slice(0, 10)} ${voting_start_time.slice(11, 16)} UTC`}
```

Comment:
is there any other less hacky way to do this?
Maybe get the date as an object and format it in another way?

Reply:
yeah, probably should use moment